.gitignore

```diff
@@ -23,3 +23,6 @@ testbin/*
 *.swp
 *.swo
 *~
+
+PROJECT
```

Comment:
why are we adding `PROJECT` to ignore?

Reply:
I think `PROJECT` should be checked in -- it wasn't there when I tried to create the collector crd and kubebuilder wouldn't work without the `PROJECT` file present